apps/rd/automation/kustomization.yaml

```diff
@@ -6,6 +6,7 @@ resources:
   - ../rd-caseworker-ref-api/demo-image-policy.yaml
   - ../rd-caseworker-ref-api-integration/image-repo.yaml
   - ../rd-caseworker-ref-api-integration/image-policy.yaml
+  - ../rd-caseworker-ref-api/demo-image-policy.yaml
```

Comment:
You have already included the same in https://github.com/hmcts/cnp-flux-config/pull/18187/files#diff-6785d5eb1baa6a835eef5b347b550da03344850a1ea823fc4a16de79a88254e1R6

Reply:
Removed that @SujeethKumarAri Thank you